src/components/ObsDetails/ObsDetails.js

```diff
@@ -241,27 +243,21 @@ const ObsDetails = ( ): Node => {
   const displayCreatedAt = ( ) => ( observation?.created_at
     ? formatObsListTime( observation.created_at ) : "" );
 
-  const displayTab = ( handlePress, testID, tabText, active ) => {
-    let textClassName = "color-gray text-xl font-bold";
-
-    if ( active ) {
-      textClassName += " color-inatGreen";
+  const tabs = [
+    {
+      id: ACTIVITY_TAB,
+      testID: "ObsDetails.ActivityTab",
+      onPress: ( ) => setTab( ACTIVITY_TAB ),
+      text: "ACTIVITY"
```

Comment:
Also here, like in other PR, we would use the translated strings here: `t( "ACTIVITY" )`

Reply:
Ah, I just see now that you have added the tranlation in the Tabs component directly. I haven't really realized then but I think I would prefer the Tabs component not to have the translation function call, and instead pass in the string already translated.
Also, the default testID should probably use the id string instead of title.